Assets/js/header.js

```diff
@@ -0,0 +1,26 @@
+const headerFn = () => {
+    const burgerBtns = document.querySelectorAll(".burger");
+    const closeIcons = document.querySelectorAll(".close-icon");
+    const menuItems = document.querySelector(".header_wrapper_menu");
+
+    const openMenu = () => {
+        burgerBtns.forEach(burgerBtn => {
+            burgerBtn.onclick = () => {
+                menuItems.classList.add("is-active");
```

Comment:
This is great. I often see developers change styles directly in javascript instead of adding/removing classes. When we use classes, it's much easier to track down where to make the changes for updates.

Reply:
Thanx!